src/Arcus.Observability.Telemetry.Serilog.Sinks.ApplicationInsights/Converters/OperationContextConverter.cs

```diff
@@ -31,5 +32,41 @@ public class OperationContextConverter
                 telemetryEntry.Context.Operation.ParentId = operationParentId;
             }
         }
+
+        /// <summary>
+        /// Enrich the given <paramref name="telemetryEntry"/> with the operation name.
+        /// </summary>
+        /// <param name="telemetryEntry">The telemetry instance to enrich.</param>
+        public void EnrichWithOperationName<TEntry>(TEntry telemetryEntry) where TEntry : ITelemetry, ISupportProperties
+        {
```

Comment:
Just a nit on the naming, can you rename things such as `r` to `requestTelemetry` please?
We don't tend to use abbreviations

Reply:
Changed, see https://github.com/arcus-azure/arcus.observability/pull/229/commits/ece70efeabf862f09c7811c5ebae8ac38a275aa1